BuildMacOSUniversalBinary.py

```diff
@@ -0,0 +1,278 @@
+            lipo(newpath0, newpath1, new_dst_path)
+
+    # loop over files in src1 and copy missing things over to dst
+    for newpath1 in glob.glob(src1+"/*"):
+        filename = os.path.basename(newpath0)
+        newpath0 = os.path.join(src0, filename)
+        new_dst_path = os.path.join(dst, filename)
+        if not os.path.exists(newpath0) and not os.path.islink(newpath1):
+            shutil.copytree(newpath1, new_dst_path)
+
+    # fix up symlinks for path0
+    for newpath0 in glob.glob(src0+"/*"):
+        filename = os.path.basename(newpath0)
+        new_dst_path = os.path.join(dst, filename)
+        if os.path.islink(newpath0):
+            relative_path = os.path.relpath(os.path.realpath(newpath0), src0)
+            os.symlink(relative_path, new_dst_path)
+    # fix up symlinks for path1
+    for newpath1 in glob.glob(src1+"/*"):
+        filename = os.path.basename(newpath1)
+        new_dst_path = os.path.join(dst, filename)
+        newpath0 = os.path.join(src0, filename)
+        if os.path.islink(newpath1) and not os.path.exists(newpath0):
+            relative_path = os.path.relpath(os.path.realpath(newpath1), src1)
+            os.symlink(relative_path, new_dst_path)
+
+
+def build(config):
+    """
+    Builds the project with the parameters specified in config.
+    """
+
+    print("Building config:")
+    print(json.dumps(config, indent=4))
+
+    dst_app = config["dst_app"]
+    # Configure and build single architecture builds for each architecture
+    for arch in ARCHITECTURES:
+        # Create build directory for architecture
+        if not os.path.exists(arch):
+            os.mkdir(arch)
+        # Setup environment variables for build
+        envs = os.environ.copy()
+        envs['PKG_CONFIG_PATH'] = config[arch+"_pkg_config_path"]
+        envs['Qt5_DIR'] = config[arch+"_qt5_path"]
+        envs['CMAKE_OSX_ARCHITECTURES'] = arch
+
+        subprocess.check_call([
+                'arch', '-'+arch,
+                'cmake', '../../', '-G', 'Xcode',
+                '-DCMAKE_OSX_DEPLOYMENT_TARGET='
+                + config[arch+"_mac_os_deployment_target"],
+                '-DMACOS_CODE_SIGNING_IDENTITY='
+                + config['codesign_identity'],
+                '-DMACOS_CODE_SIGNING_IDENTITY_UPDATER='
+                + config['codesign_identity'],
+                '-DMACOS_CODE_SIGNING="ON"'
+            ],
+            env=envs, cwd=arch)
+
+        # Build project
+        subprocess.check_call(['xcodebuild',
+                               '-project', 'dolphin-emu.xcodeproj',
+                               '-target', config["build_target"],
+                               '-configuration', 'Release'], cwd=arch)
+
+    # Source binary trees to merge together
+    src_app0 = ARCHITECTURES[0]+"/Binaries/release"
+    src_app1 = ARCHITECTURES[1]+"/Binaries/release"
+
+    if os.path.exists(dst_app):
+        shutil.rmtree(dst_app)
+
+    os.mkdir(dst_app)
```

Comment:
Both comments below can be merged into a single one and put above the `os.mkdir(dst_app)` code:
```python
# Create and codesign the universal binary
```

Reply:
Okay, I will make this change.